in_toto/util.py

```diff
@@ -1,62 +1,53 @@
 import sys
 import getpass
+from enum import Enum
```

Comment:
The `enum` module was [added in Python 3.4](https://docs.python.org/3/library/enum.html).  It isn't available in Python 2 unless you use one of its backports (e.g., enum34).

I think this import statement is only working because enum34 was added as a third-party dependency (via cryptography?).

You should add `enum34` as dependency for `in-toto`, especially if we ever decide to make 'cryptography' an optional dependency. 

Another option is to avoid Enum entirely, and just use a simple list or dictionary.  We should store constants of this sort in gpg/constants.py, or similar.